auction-server/src/opportunity/service/get_quote.rs

```diff
@@ -129,13 +136,13 @@ impl Service<ChainTypeSvm> {
             ),
             chain_id:       quote_create.chain_id.clone(),
             sell_tokens:    vec![entities::TokenAmountSvm {
-                token:  input_mint,
```

Comment:
isn't the input token in this case that which the searcher is providing? https://github.com/pyth-network/per/blob/240b2ba21076ca0a1af86af23c0e34271d499c96/contracts/svm/programs/express_relay/src/lib.rs#L180-L182